lib/core/src/lib/viewer/components/pdf-viewer.component.ts

```diff
@@ -302,33 +303,37 @@ export class PdfViewerComponent implements OnChanges, OnDestroy {
 
             let scale: number;
             switch (this.currentScaleMode) {
-                case 'init':
+                case 'init': {
                     scale = this.getUserScaling();
                     if (!scale) {
                         scale = this.autoScaling(pageHeightScale, pageWidthScale);
                     }
                     break;
-                case 'page-actual':
+                }
+                case 'page-actual': {
                     scale = 1;
                     break;
-                case 'page-width':
+                }
+                case 'page-width': {
                     scale = pageWidthScale;
                     break;
-                case 'page-height':
+                }
+                case 'page-height': {
                     scale = pageHeightScale;
                     break;
-                case 'page-fit':
+                }
+                case 'page-fit': {
```

Comment:
small thing as you are touching this code - this case is the same like case for 'init' (306th line) so it can be: 
```ts
case 'init':
case 'page-fit': {
scale = this.getUserScaling();
    if (!scale) {
        scale = this.autoScaling(pageHeightScale, pageWidthScale);
    }
    break;
}

Reply:
good point, thanks, fixed